src/main/java/com/example/onlinebookstore/model/Book.java

```diff
@@ -39,4 +44,12 @@ public class Book {
 
     @Column(nullable = false)
     private boolean isDeleted = false;
+
```

Comment:
Be attentive to Data

Reply:
you mean that I need to put @ToString.Exclude?